### DISCUSSION TITLE
Tuning SSC

### DISCUSSION THREAD

**Discussion by ta55el** [chromiecraft/chromiecraft#6285] [Suggestions]
I suggest to give raid buffs at least 400% HP and 20% dmg increased for all mobs and bosses in instnance. Why?
In current gear and wotlk talents we have at least twice raid dps more and mobs has more than twice less HP than it was back in the day of TBC. Dmg seems little low comparing some spells but tanks get less dmg cuz of better talents that reduces dmg and our healers also are much better and more versatile. You could decrease buffs some part every week if it would be too hard for rest of the server.

https://www.wowhead.com/tbc/npc=21221/coilfang-beast-tamer
430k HP
On our server they have 176k HP
We are doing at least twice dps so 176kx4=704k HP
Similar with other mobs
https://www.wowhead.com/tbc/npc=21251/underbog-colossus
https://wowwiki-archive.fandom.com/wiki/Underbog_Colossus?oldid=770610
886k on TBC classic and 620 on TBC retail but on our server they have 387k

**Comment by shaszun** [comment]
Current stats are retail, that's why they are low HP. But for a tuning, My opinion is at least 2x of the original TBC values for trash. Bosses would require to be checked 1 by 1 to find a suitable scaling.

**Reply by ta55el** [reply]
Current stats are retail from wotlk after nerfs. Best would be to scale fight duration on bosses that they would not last 2-3 min, but at least 5min+. Guilds are doing 40k+ raid dps and mana managment is also much better with wotlk talents. Would be nice to test SSC if staff will add buffs 400% HP and 20% dmg for whole instance and after testing we could make proper corrections.
